controllers/users.js

```diff
@@ -143,11 +144,33 @@ const postUserPicture = async (req, res) => {
   }
 }
 
+/**
+ * Post user profile picture
```

Comment:
Please add comment for `updateChainCode`

Reply:
Right. Thanks for pointing it out.